Cargo.toml

```
@@ -33,13 +33,13 @@ pretty_env_logger = "0.4.0"
 rand = "0.8.4"
 reqwest = { version = "0.11.6", features = ["blocking", "json"] }
 serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0.73"
+serde_json = "1.0.78"
 serial_test = "0.5.1"
 sha2 = { version = "0.9.8" }
 stringreader = "0.1.1"
 strum = "0.22.0"
 strum_macros = "0.22.0"
-synapse-rpc = "0.1.20"
+synapse-rpc = { git = "https://github.com/tiainen/synapse", branch = "pyrsia-synapse" }
```

Once https://github.com/pyrsia/synapse/pull/1 is merged, this should be changed:

```suggestion
synapse-rpc = { git = "https://github.com/pyrsia/synapse", branch = "master" }
```